RobotCrawler/src/main/java/server/service/BookFinderController.java

```diff
@@ -0,0 +1,37 @@
+package server.service;
+
+import model.BookData;
+import model.EBookCategory;
+import model.response.EBooksResponse;
+import model.response.ResponseBase;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import robot.ebooks.EBooksRobotManager;
+
+import java.util.EnumMap;
+import java.util.List;
+
+@RestController
+@RequestMapping("/robot")
+public class BookFinderController {
+
+    @Autowired
+    @Qualifier("ebooks-robot-manager")
```

Comment:
By qualifier i avoid the conflict in the future. I always like to name everything even if i use it in one place.